docs/UserGuide.md

```diff
@@ -426,24 +477,42 @@ starts on 30 Jan 2022 and ends on 23 Nov 2022 to the 1st student in the displaye
 
 Edits the specified lesson of the specified student in TAB with the indicated changes for specified fields.
 
-Format: `ledit INDEX LESSON_INDEX [recurring/[END_DATE]] [date/START_DATE] [time/TIMERANGE] [subject/SUBJECT] [rates/LESSON_RATES] [f/OUTSTANDING_FEES] [hw/HOMEWORK]… [cancel/CANCEL_DATE]… [uncancel/UNCANCEL_DATE]…​`
+Additional fields to the ones in [Managing lessons](#managing-lessons) are listed below.
+
+Field | Prefix | Constraints | Examples |
+------------------|-------|-----------| --------|
+Cancelled date |`cancel/`| Same constraints as other date fields.<br/> The date to be cancelled must be a valid lesson date.<br>e.g. If the start date of a recurring lesson is `1 Oct 2021`, you can cancel `8 Oct 2021` but not `2 Oct 2021`.|`cancel/20 jan 2022`|
+Uncancelled date |`uncancel/` | Same constraints as other date fields.<br/>The date to be uncancelled must be a date that has already been cancelled.| `uncancel/20 jan 2022`|
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
```suggestion
<div markdown="block" class="alert alert-warning">:exclamation: **Caution:** <br>
```

Reply:
Actually, I think span looks better? Block seems to have some extra spaces at the bottom

## Span

![image](https://user-images.githubusercontent.com/62977276/140271829-85925d6e-b313-40e4-8157-9b6844f1c513.png)

## Block

![image](https://user-images.githubusercontent.com/62977276/140271860-76da1046-69d6-4a7f-ac3a-b6b2a9d0b2c1.png)

Reply:
Without the <br> , with block 

![image](https://user-images.githubusercontent.com/62977276/140272021-da643ff1-302d-4d41-ad38-dd028c80fb46.png)

Without <b>, with span
![image](https://user-images.githubusercontent.com/62977276/140272084-5c953d4d-9812-4cce-8a69-6993b7534f7b.png)

Reply:
ohh i tot this is the part where there were supposed to be bullet points but was not displayed properly

Reply:
Oh yeah, I changed it so the bullet points are outside the caution block so its up to the users whether they want to read the cases but we will still warn them.